docs/4_connectivity.md

```diff
@@ -179,7 +179,7 @@ int main(int, char**) {
 
 ### Program logic
 
-The code sample above does not do much, except for setting up the connection. We can now define some logic for this program:
+The code sample above does not do much except for setting up the connection. We can now define some logic for this program:
```

Comment:
Query: Could we rewrite this sentence? It's a negative view that the code sample doesn't do much. Could we rewrite it as "The code sample above sets up the connection" or something similar?